roles/ceph-ansible/defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# home directory which contains copy of files in ceph-ansible package
+home_dir: "{{ ansible_env['WORKSPACE'] | default(ansible_env['HOME']) }}"
```

Comment:
The name ``home_dir`` sounds to me, like it should be the HOME dir of the user, I would name it maybe something like ``base_dir`` (if I understand the meaning correctly).
Also where should point the ``ansible_env['WORKSPACE']``? (I wasn't able to find, what it means.)